tests/X86/SSE/UNPCKHPD.S

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017 Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+TEST_BEGIN_64(UNPCKHPDv128v128, 2)
+TEST_INPUTS(
+    0x0000000000000000, 0x0000000000000000,  // sanity check
```

Comment:
One way to get at "arbitrarily" chosen values is to use what's already pre-loaded in the `xmm`/`ymm` registers. Here is the data: https://github.com/trailofbits/remill/blob/master/tests/X86/Tests.S#L183  And `xmm`/`ymm` zero through seven are initialized with these data: https://github.com/trailofbits/remill/blob/master/tests/X86/Tests.S#L240

Otherwise this is fine, nothing to change.